package.json

```diff
@@ -179,13 +182,16 @@
     "csso": "^4.0.3",
     "electron": "^8.2.5",
     "electron-builder": "^22.6.0",
+    "electron-devtools-installer": "^3.0.0",
     "electron-notarize": "^0.3.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.0.0",
```

Comment:
Have you cross-checked that nothing breaks? Especially not the custom rules in the `.eslintrc`?

Reply:
I didn't noticed any problems, and linting still works.

Reply:
Alright, thanks!